package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@edx/studio-frontend",
-  "version": "1.5.0",
+  "version": "1.6.1",
```

Comment:
I was under the impression that when you change the minor version, the patch version gets reset to 0? Is that incorrect?

https://semver.org/#spec-item-7

Reply:
My reasoning was: 

1. bump to v1.6.0 for the preview image toggle (we never released that)
2. bump to v1.6.1 for the IE11 fix and the Mac Firefox searchfield fix

I can make it 1.6.0 if you think that makes more sense though.

Reply:
Ah, I see. That makes sense. Thanks. I'll approve it now.